Pinta.Core/Classes/Document.cs

```diff
@@ -353,25 +363,26 @@ public void ResizeImage (Size newSize, ResamplingMode resamplingMode)
 	// Rotate image 180 degrees (flip H+V)
 	public void RotateImage180 ()
 	{
-		RotateImage (180);
+		RotateImage (new (180));
```

Comment:
For these functions an explicit `new DegreesAngle (180)` might be more clear?

Reply:
An alternative approach that keeps the clarity without wrapping the value in a structure would be to name the parameter. If the parameter were named `degrees` instead of `angle` then the call site could be:
```
    RotateImage (degrees: 180);
```
Coming at it from the other direction, it seems to me that the real strength of having these separate types `RadiansAngle`, `DegreesAngle` and `RevolutionsAngle` would come from polymorphism. If you added an interface `IAngle` that specified a property getter for whatever representation is the most convenient:
```
    public interface IAngle
    {
        double Degrees { get; }
    }
```
...then each of the angle classes could provide its own implementation:
```
    public class RadiansAngle : IAngle
    {
        ...
        double IAngle.Degrees => Radians * 180.0 / Math.PI;
        ...
    }

    public class RevolutionsAngle : IAngle
    {
        ...
        double IAngle.Degrees => Revolutions * 360.0;
        ...
    }

    public class DegreesAngle : IAngle
    {
        // DegreesAngle already has a property Degrees { get; } :-)
        ...
    }
```

Then, `RotateImage` could take an `IAngle` and simply use its `.Degrees` property. This then becomes a flexible tool for any future code that needs to interact with it.

Reply:
@logiclrd this was discussed in #569 so I suggest you take a look at that one.

Reply:
I see :-)

I noticed that `GetNearestStepAngle` has another occurrence of `new (value)` in it, by the way.